public/index.html

```diff
@@ -25,7 +25,7 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <title>React Redux App</title>
-    <script id="react-dotenv" src="./env.js"></script>
```

Comment:
this won't fix the issue on testnet. all the files in public are regenerated at build time. you can test this by going to the PR deployment:

https://deploy-preview-25--quicksilver-app.netlify.app/stake/delegate